### ISSUE TITLE
ShellcmdLrms does not correctly report exit status of programs that were killed

### ISSUE THREAD

**Issue by riccardomurri** [uzh/gc3pie#645]
As reported by @DavidDreher, a MATLAB script that runs on a VM and is killed by the Linux OOM killer (signal 9), still is reported as having "exit status 0" by GC3Pie.

**Comment by riccardomurri** [comment]
It looks like GNU `time` does output "0" as exit code for the a program that's killed by signal 9; here is code to reproduce this behavior::

```c
$ cat xxx.c
#include <unistd.h>
#include <signal.h>
#include <sys/types.h>

int
main(int argc, char** argv)
{
  sleep(1);
  kill(getpid(), 9);
}

$ gcc xxx.c -o xxx

$ /usr/bin/time --format 'exitcode=%x' ./xxx
Command terminated by signal 9
exitcode=0
FAIL: 137
```

Still, `time` *does* exit with code (128 + signal nr.) so that information gets overwritten somewhere in `ShellcmdLrms` -- quite likely, we trusted the `%x` to be the same as `time`'s exit code.

**Comment by riccardomurri** [comment]
This has been fixed in GC3Pie "master", commit  98baa83
